tests/integ/test_udf.py

```diff
@@ -1381,10 +1381,10 @@ def test_udf_parallel(session):
     reason="numpy and pandas are required",
 )
 def test_add_packages(session):
-    session.add_packages(["numpy==1.20.1", "pandas==1.3.5", "pandas==1.3.5"])
+    session.add_packages(["numpy==1.23.5", "pandas==1.5.3", "pandas==1.5.3"])
```

Comment:
hmm the old versions are not supported anymore?

Reply:
The old packages do not have a py310 build. I don't think our test has a requirement on using the old packages so I updated it such that we don't need to complicate the tests with a python version -> package version mapping.